packages/components/src/components/text-field-wrapper/text-field-wrapper-styles.ts

```diff
@@ -63,6 +69,15 @@ export const getComponentCss = (
           ...(isNumber && {
             MozAppearance: 'textfield', // hides up/down spin button for Firefox
           }),
+          ...((isCalendar || isTime) && {
+            // for native placeholder color in safari, background ahs to be a special value, `transparent` or `rgba(0,0,0,0)` won't work
+            // this works nice for `theme="light"`, but for `theme="dark"` placeholder appears blue which is still better
+            // than having invisible dots or colons for data/time or not seeing the value at all after selection
+            '@supports (-webkit-hyphens: none)': {
```

Comment:
What is this media query for? Is it a Safari only media query?

Reply:
Added the link to the original source.